<launch>
    <arg name="rviz_config" default="$(find rrt_planner)/cfg/rrt_planner.rviz"/>
    <arg name="map_file" default="$(find rrt_planner)/cfg/map.yaml"/>

    <rosparam command="load" file="$(find rrt_planner)/cfg/config.yaml"/>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(arg rviz_config)"/>
    <node name="map_server" pkg="rrt_planner" type="map_server.py" args="$(arg map_file)" output="screen"/>
    <node type="rrt_planner_node" name="rrt_planner" pkg="rrt_planner" output="screen"/>
</launch>